EasyLearn/app/views/player/index.html.erb

```diff
@@ -0,0 +1,52 @@
+<%= stylesheet_link_tag "player", "data-turbo-track": "reload" %>
+
+  <nav>  
+    <h3 >Logo</h3>
+    <div class="menu">
+      <p>My events</p>
+      <div class="user-option">
+        <p class="username">username</p>
+      </div>  
+    </div>
+  </nav>
```

Comment:
I think i saw this nav in the past PR of @Joshsam31, and he made an partial of it, you could change this code and use that partial.
Send me a message if you have any questions🤓

Reply:
Totally agree with you @xFerchoVJ

Reply:
Done, changed the navbar